python3.8/cookiecutter-aws-sam-efs-python/{{cookiecutter.project_name}}/README.md

```diff
@@ -57,13 +57,17 @@ To simplify troubleshooting, SAM CLI has a command called `sam logs`. `sam logs`
 
 You can find more information and examples about filtering Lambda function logs in the [SAM CLI Documentation](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-logging.html).
 
-## Unit tests
+## Tests
 
-Tests are defined in the `tests` folder in this project. Use PIP to install the [pytest](https://docs.pytest.org/en/latest/) and run unit tests.
+Tests are defined in the `tests` folder in this project. Use PIP to install the test dependencies and run tests.
 
 ```bash
-{{ cookiecutter.project_name }}$ pip install pytest pytest-mock --user
-{{ cookiecutter.project_name }}$ python -m pytest tests/ -v
+{{ cookiecutter.project_name }}$ pip install -r tests/requirements.txt --user
```

Comment:
Not directly relevant to this PR, but I'm not a fan of prefixing the project name here.

I haven't seen any README that would use such a pattern, and in its current state if the project name is `a`, it'll show:

```
a$ python -m pytest tests/ -v
```

Which looks plain confusing (that there's no space certainly doesn't help).

Instead I'd prefer to only have the prompt character (e.g. `$`), or even better, the commands verbatim, so that users can quickly triple-click, copy-and-paste and run the commands as-is (without having to tediously copy one-by-one, making sure not to include the prompt).

Reply:
Good idea! Let's have a followup PR